homeassistant/components/totalconnect/quality_scale.yaml

```diff
@@ -0,0 +1,65 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
```

Comment:
I think the config flow tests itself can use some love though:
- `test_user` can be merged with the next one `test_user_show_locations`
- Ideally we mock out the library at where we use them and not internals (we currently patch `homeassistant.components.totalconnect.TotalConnectClient.request`)
- In `test_user_show_locations` we should also assert the set unique_id and the stored data
- `test_abort_if_already_setup`: The mock for the library is not needed here as we abort before we call the library
- `test_login_failed`: Let's finish this flow by repatching the library in a working state to also test that the config flow is able to recover from such error
- `test_no_locations` line 209 `await hass.async_block_till_done()` is not needed (this is just a remark)
- So apparently the auto_bypass is used to set a variable on the client library. Is that propogated to the device? Should that be a switch entity?

Reply:
set to `todo`